.github/workflows/lint.yml

```
@@ -57,5 +57,5 @@ jobs:
       - name: Install dependencies
         run: bun install
 
-      - name: Validate commit messages
-        run: bunx commitlint --from ${{ github.event.pull_request.base.sha }} --to ${{ github.event.pull_request.head.sha }} --verbose
+      - name: Validate PR title (for squash merge)
+        run: echo "${{ github.event.pull_request.title }}" | bunx commitlint --verbose
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)